include/sea_dsa/Graph.hh

```diff
@@ -520,6 +520,14 @@ public:
     m_nodeType.heap = v;
     return *this;
   }
+  Node &setIncomplete(bool v = true) {
```

Comment:
in addition to `setIncomplete` you need `markIncomplete`. If a node is incomplete, then any node reachable from it is incomplete as well. `setIncomplete` just sets the flag. `markIncomplete` traverses the graph and propagates the marking